app/views/field_of_operation/show.html.erb

```diff
@@ -27,19 +27,20 @@
         } %>
       </div>
     <% end %>
-    <div class="govuk-grid-column-two-thirds govuk-!-margin-bottom-7">
+    <div class="govuk-grid-column-two-thirds govuk-!-margin-bottom-8">
       <section id="field-of-operation">
         <% unless @presenter.description.blank? %>
           <%= render "govuk_publishing_components/components/heading", {
             text: t("formats.fatality_notice.field_of_operation"),
             margin_bottom: 4,
           } %>
           <%= render "govuk_publishing_components/components/govspeak", {
+              margin_bottom: 8,
             } do %>
               <%= @presenter.description %>
           <% end %>
         <% end %>
-        <div class="govuk-!-margin-top-7 govuk-!-padding-bottom-3">
+        <div>
```

Comment:
If this div has no classes, is it still needed?

Reply:
Thanks, have removed it 👍